temporalcli/commands.go

```diff
@@ -229,10 +229,13 @@ func UnmarshalProtoJSONWithOptions(b []byte, m proto.Message, jsonShorthandPaylo
 	return opts.Unmarshal(b, m)
 }
 
-func (c *CommandContext) populateFlagsFromEnv(flags *pflag.FlagSet) error {
+// Set flag values from environment file & variables. Returns a map of flags->values which were
+// overridden by environment variables.
```

Comment:
```suggestion
// Set flag values from environment file & variables.
```

Doesn't return a map (though may read better if it did return a list of changes instead of a logger callback, but this is a private function so it's fine)

Reply:
Yeah forgot to change this, I didn't want to make a new struct type just to return what was altered